addons/website_slides/views/website_slides_templates_lesson.xml

```diff
@@ -174,22 +174,18 @@
                             </div>
                         </a>
                     </div>
-                    <ul t-if="aside_slide._has_additional_resources() or aside_slide.question_ids" class="o_wslides_lesson_aside_list_links list-group mb-1 list-unstyled fw-light">
-                        <t t-if="can_access_channel">
-                            <t t-foreach="aside_slide.slide_resource_ids" t-as="resource" t-if="resource.resource_type == 'url'">
-                                <li class="ps-3">
-                                    <a t-if="can_access" t-att-href="resource.link" target="new" class="text-decoration-none small">
-                                        <i class="fa fa-link"/><span t-field="resource.name"/>
-                                    </a>
-                                </li>
-                            </t>
-                            <div class="o_wslides_js_course_join ps-3" t-if="aside_slide._has_additional_resources()">
-                                <li t-foreach="aside_slide.slide_resource_ids" t-as="resource" t-if="resource.resource_type == 'file'">
-                                    <a t-att-href="resource.download_url" class="text-decoration-none small">
-                                        <i class="fa fa-download"/><span t-field="resource.name"/>
-                                    </a>
-                                </li>
-                            </div>
+                    <ul t-if="aside_slide.sudo().slide_resource_ids or aside_slide.question_ids" class="o_wslides_lesson_aside_list_links list-group mb-1 list-unstyled fw-light">
+                        <t t-if="can_access_channel" t-foreach="aside_slide.slide_resource_ids" t-as="resource">
+                           <li t-if="resource.resource_type == 'url'" class="ps-3">
```

Comment:
I think we can move the t-if/t-else check on the `<a>` tag level as both have the same li tag it looks like?